proprietary/tokens/src/brand/ams/typography.tokens.json

```diff
@@ -0,0 +1,39 @@
+{
+  "ams": {
+    "brand": {
+      "typography": {
+        "font-family": {
+          "body-text": { "value": "'Amsterdam Sans', Arial, sans-serif" },
+          "headings": { "value": "'Amsterdam Sans', Arial, sans-serif" },
+          "monospace": { "value": "monospace" }
```

Comment:
Might be too soon to add this.

Reply:
If we add a `<code>` component it might come in handy.